pkg/config/cacheentrryentity.go

```diff
@@ -0,0 +1,66 @@
+package config
+
+import (
+	"crypto/md5"
+	"fmt"
+	"time"
+
+	"github.com/kyma-incubator/reconciler/pkg/db"
+)
+
+type CacheEntryEntity struct {
+	ID       int64     `db:"readOnly"`
+	Label    string    `db:"notNull"`
+	Cluster  string    `db:"notNull"`
+	Data     string    `db:"notNull"`
+	Checksum string    `db:"notNull"`
+	Created  time.Time `db:"readOnly"`
+}
+
+func (ce *CacheEntryEntity) String() string {
+	return fmt.Sprintf("Label=%s,Cluster=%s,Checksum=%s,CreatedOn=%s",
+		ce.Label, ce.Cluster, ce.checksum(), ce.Created)
+}
+
+func (ce *CacheEntryEntity) New() db.DatabaseEntity {
+	return &CacheEntryEntity{}
+}
+
+func (ce *CacheEntryEntity) checksum() string {
+	if ce.Checksum == "" && ce.Data != "" {
+		ce.Checksum = ce.md5()
+	}
+	return ce.Checksum
+}
+
+func (ce *CacheEntryEntity) md5() string {
+	md5 := md5.Sum([]byte(ce.Data)) //nolint - MD5 is used for change detection, not for encryption
```

Comment:
Maybe you can come up with different name than md5 for that variable? It covers the md5 import.

Reply:
Definitely, var is now renamed!